tasks/lib/helpers/git.js

```diff
@@ -52,7 +52,7 @@ function checkout(grunt, dir, branchName, done) {
 
 function merge(grunt, dir, mergeFromBranch, msgPrefix, msg, done) {
 
-  exec('git merge --no-ff -m "' + msgPrefix + ' ' + msg + '" ' + mergeFromBranch, {
+  exec('git merge --no-ff --strategy-option theirs -m "' + msgPrefix + ' ' + msg + '" ' + mergeFromBranch, {
```

Comment:
Good question. It might make sense to make that optional.

What does `theirs` refer to in this case? The hotfix branch or the master branch? I guess it has to be hotfix branch, right?